framework/PostProcessors/ParetoFrontierPostProcessor.py

```diff
@@ -65,11 +67,21 @@ def _handleInput(self, paramInput):
       @ In, paramInput, ParameterInput, the already-parsed input.
       @ Out, None
     """
+    self.valueLimit = None
+    self.costLimit  = None
```

Comment:
move these two variables to the ```__init__``` method and provide some explanation for them.

Reply:
Done